temporal/api/cloud/sink/v1/message.proto

```diff
@@ -0,0 +1,109 @@
+syntax = "proto3";
+
+package temporal.api.cloud.sink.v1;
+
+import "google/protobuf/timestamp.proto";
+
+
+option go_package = "go.temporal.io/api/cloud/sink/v1;sink";
+option java_package = "io.temporal.api.cloud.sink.v1";
+option java_multiple_files = true;
+option java_outer_classname = "MessageProto";
+option ruby_package = "Temporalio::Api::Cloud::Sink::V1";
+option csharp_namespace = "Temporalio.Api.Cloud.Sink.V1";
+
+message ExportSinkSpec {
+    // The name of the export sink.
+    string name = 1;
+
+    // A flag indicating whether the export sink is enabled or not.
+    bool enabled = 2;
+
+    // The receiving sink type for export.
+    ExportDestinationType destination_type = 3;
+    
+    // The S3 configuration details when destination_type is S3.
+    S3Spec s3_sink = 4;
+
+    // The GCS configuration details when destination_type is GCS.
+    GCSSpec gcs_sink = 5;
+
+    enum ExportDestinationType {
+        EXPORT_DESTINATION_TYPE_UNSPECIFIED = 0;
+        EXPORT_DESTINATION_TYPE_S3 = 1;
+        EXPORT_DESTINATION_TYPE_GCS = 2;
+    }
+}
+
+
+message ExportSink {
```

Comment:
Will this sink be used for anything other then for namespace exports? If no, we should move it to the namespace package, i think.

Reply:
Another thing is sink has its only internal storage. So it's not depends on namespace table. It's a sub resource of namespace. Thus, I'm hesitated to merge to namespace. 

Do you think we could move the sink folder under namespace folder?

Reply:
You could move it to a sub folder under the namespace folder. Just feels like it will be really thin for it to be its own package. Also we will be introducing a new pattern where we have sub packages (all our current packages are at root).

But we clearly need to move the   ExportSink and ExportSinkSpec to the namespace package (or namespace subpackage).
Just to make sure we are on the same page the `S3Spec` and `GCSSpec` will remain here in the sink package, correct?

Reply:
The only thing is this change has already landed in saas-proto. Can we have a sync on this and see how strong you feel this needs to change?